### PR TITLE
Update readme for autocmd support

### PR DESCRIPTION
**Pull request by rakanzaidm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,25 @@ require('session_manager').setup({
   max_path_length = 80,  -- Shorten the display path if length exceeds this threshold. Use 0 if don't want to shorten the path at all.
 })
 ```
+
+## Autocommands
+
+The plugin supports user configurable autocmd using vim's very own User event feature. You can specify commands to be executed automatically after saving or loading a session. Use these like any other built in vim autocmd events.
```
Comment (Shatur) [inline README.md:50]: Let's make it shorter:

```suggestion
You can specify commands to be executed automatically after saving or loading a session using the following autocmds:
```

File: README.md [hunk 2]
```diff
@@ -44,3 +44,25 @@ require('session_manager').setup({
...
+
+| Event                | Description                        |
+| -------------------- | ---------------------------------- |
+| User SessionSavePost | Executed after a session is saved  |
+| User SessionLoadPost | Executed after a session is loaded |
```
Comment (Shatur) [inline README.md:55]: Custom autocmds are always user-defined:
```suggestion
| SessionSavePost | Executed after a session is saved  |
| SessionLoadPost | Executed after a session is loaded |
```

File: README.md [hunk 3]
```diff
@@ -44,3 +44,25 @@ require('session_manager').setup({
   max_path_length = 80,  -- Shorten the display path if length exceeds this threshold. Use 0 if don't want to shorten the path at all.
 })
 ```
+
+## Autocommands
+
+The plugin supports user configurable autocmd using vim's very own User event feature. You can specify commands to be executed automatically after saving or loading a session. Use these like any other built in vim autocmd events.
+
+| Event                | Description                        |
```
Comment (Shatur) [inline README.md:52]:
```suggestion
| Autocmd                | Description                        |
```

**Review by Shatur — CHANGES_REQUESTED** [review]

**Comment by rakanzaidm** [comment]
alrighty!!

**Comment by Shatur** [comment]
Could you also format added table after removing `User`?

**Comment by rakanzaidm** [comment]
oh wait right i missed that one HAHA on my way

**Pull request merged**